### PR TITLE
Add model face toolbox (LResNet, ir50m1sm and ir50asia models)

### PR DESCRIPTION
**Pull request by AndreaMacarulla**
Models extracted from https://github.com/shaoanlu/face_toolbox_keras.

### CODE REVIEW

**Comment by JeroenMandersloot** [comment]
Could you add some unit tests to `test_architectures.py` and `test_finetuning.py` to see if everything works correctly? You should be able to just copy some of the existing tests and modify them slightly to use these newly added architectures :)

File: insightface/basemodels/ir50asia.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+from keras.layers import *
```
Comment (JeroenMandersloot) [inline insightface/basemodels/ir50asia.py:1]: Importing from keras directly might give issues with finetuning, import from tensorflow.keras instead if possible.

File: insightface/basemodels/ir50asia.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+from keras.models import Model
```
Comment (JeroenMandersloot) [inline insightface/basemodels/ir50asia.py:2]: Same as above

File: insightface/basemodels/ir50m1sm.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+from keras.layers import *
+from keras.models import Model
```
Comment (JeroenMandersloot) [inline insightface/basemodels/ir50m1sm.py:2]: `tensorflow.keras` instead of `keras`

File: insightface/verifier/face_verifier.py [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+#from . import umeyama
+
+FILE_PATH = Path(__file__).parent.resolve()
+
+import sys
+sys.path.append(str(FILE_PATH.parent.parent.parent.resolve()))
+from utils.umeyama import umeyama
+
+class FaceVerifier():
```
Comment (JeroenMandersloot) [inline insightface/verifier/face_verifier.py:18]: Do we use this class (or this file for that matter)?

File: insightface/weights/weights.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Weights go here.
```
Comment (JeroenMandersloot) [inline insightface/weights/weights.md:1]: Can we remove this file?

Comment (JeroenMandersloot) [reply insightface/weights/weights.md:1]: Which weights are stored here exactly? And should we add this folder to the gitignore?

**Comment by JeroenMandersloot** [comment]
I think the embeddings of the models still have to be L2-normalized, which is necessary for finetuning them effectively.

**Comment by JeroenMandersloot** [comment]
Add `insightface/checkpoints/` to the `.gitignore`?
